changelog.md

```diff
@@ -1,3 +1,29 @@
+# 0.2.1 Release
+This is the second release for the Ternoa Testnet.
+The two major features are the two new pallets capsule and associated-accounts.
```

Comment:
Pallets is way too technical term for this changelog segment. Try to rephrase it so that a grandma could understand it.

Reply:
✅